app/models/application_record.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ActiveRecord::Base
+  self.abstract_class = true
+end
```

Comment:
hrmmm, this is supposed to be coming from foreman.   I wonder if the migration that failed needs to be changed to not use the "Katello::Product" class.